src/components/Person/Person/Person.jsx

```diff
@@ -0,0 +1,28 @@
+export const Person = ({
+  person: {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  }
+}) => (
+  <>
+    <section className="Person">
```

Comment:
```suggestion
  <section className="Person">
```
React fragment is redundant here. You already have a container for person